components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/model/BasicOrganization.java

```diff
@@ -18,11 +18,15 @@
 
 package org.wso2.carbon.identity.organization.management.service.model;
 
+import java.io.Serializable;
+
 /**
  * This class represents the model of an organization.
  * This model is used for pagination purposes in get organizations API call.
  */
-public class BasicOrganization {
+public class BasicOrganization implements Serializable {
```

Comment:
Why we make the model serializable ? We should only serialize cache keys and entries

Reply:
Fixed with https://github.com/wso2/identity-organization-management-core/pull/194/commits/89824b49512ce1bbde2edc2412f0f790faa0df69